Docs/sample_bin/admin_scripts_setup

```diff
@@ -4,10 +4,10 @@ SUBMITTY_VAR_DIR=/var/local/submitty
 SUBMITTY_REPOSITORY=/usr/local/submitty/GIT_CHECKOUT_Submitty
 
 # Copy account maintenance scripts unless they exist
-mkdir -p $SUBMITTY_INSTALL_DIR/bin
+mkdir -p ${SUBMITTY_INSTALL_DIR}/bin
 mkdir -p /root/bin
-mkdir -p $SUBMITTY_VAR_DIR/instructors
-mkdir -p $SUBMITTY_VAR_DIR/bin
+mkdir -p ${SUBMITTY_VAR_DIR}/instructors
```

Comment:
Elsewhere we call the SUBMITTY_DATA_DIR.  (right? same thing?)
We should standardize.